packages/build-utils/src/lambda.ts

```diff
@@ -123,7 +137,10 @@ export class Lambda {
         '"regions" is not a string Array'
       );
     }
+
     this.type = 'Lambda';
+    this.operationType = operationType;
+    this.bundledFunctions = bundledFunctions;
```

Comment:
**question:** is there a test I should write around this?

It doesn't sound super useful to test that the constructor can assign these values, but that's about the only logic in this PR.